docs/cue/reference/components.cue

```diff
@@ -59,7 +59,7 @@ components: {
 		// undergoes a name change.
 		alias?: !=""
 
-		// `type` is the component identifier. This is set automatically.
+		// `type` is the component name. This is set automatically.
```

Comment:
```suggestion
		// `type` is the component type. This is set automatically.
```

I think this was previously incorrectly saying identifier. It should match the component type rather than its name.

Reply:
Makes more sense :)